packages/cfd/build/config.js

```diff
@@ -0,0 +1,59 @@
+const path = require('path');
+const stylelintFormatter = require('stylelint-formatter-pretty');
+const { IS_RELEASE } = require('./constants');
+// const { transformContentUrlBase } = require('./helpers');
+
+const generateSWConfig = () => ({
+    importWorkboxFrom: 'local',
+    cleanupOutdatedCaches: true,
+    exclude: [/CNAME$/, /index\.html$/, /404\.html$/],
+    skipWaiting: true,
+    clientsClaim: true,
+});
+
+const htmlOutputConfig = () => ({
+    template: 'index.html',
+    filename: 'index.html',
+    minify: !IS_RELEASE
+        ? false
+        : {
+              collapseWhitespace: true,
+              removeComments: true,
+              removeRedundantAttributes: true,
+              useShortDoctype: true,
+          },
+});
+
+const htmlInjectConfig = () => ({
```

Comment:
I suggest removing htmlInjectConfig() because it's not used in [packages/cfd/build/constants.js](https://github.com/binary-com/deriv-app/pull/5036/files#diff-db2f11e8b5dbf94d1ec1c725ef197a705c21e166506360fdce87e99ddf63426e) same like I was recommended to do it with trader package in my PR #5063 recently